Projectiles/Melee/CosmicShivAura.cs

```diff
@@ -44,7 +44,7 @@ public override void OnSpawn(IEntitySource source)
         public override void AI()
         {
             // Unexist with target
-            if (!target.active || target is null) {
+            if (!target?.active == true || target is null) {

```

Comment:
By flipping these cases around you don't need to guard against target being null twice.
```suggestion
            if (target is null || !target.active) {
```